Documentation/policy/language.rst

```diff
@@ -202,13 +203,11 @@ by the selected endpoints.
 The following entities are defined:
 
 host
-  The local host serving the endpoint
-
+    The local host serving the endpoint.
```

Comment:
We have to correct this. Given the datapath behavior, it can also include remote nodes in the same mesh.

Reply:
Do you mean, adjust the data path behaviour or document?

Reply:
Documenting.

Reply:
Yes, documenting